libconcurrent/primitives/primitives.c

```diff
@@ -13,13 +13,20 @@ extern __thread int64_t __executed_faa;
 
 #ifdef __NO_GCC_SPARC__
 inline static uint32_t __bitSearchFirst32(uint32_t v) {
-    register uint32_t r;     // The result of log2(v) will be stored here
+    register uint32_t r; // The result of log2(v) will be stored here
     register uint32_t shift;
 
-    r =     (v > 0xFFFF) << 4; v >>= r;
```

Comment:
This needs fix.
It makes the code unreadable

Reply:
Given that this is a very small part of the code that the styling is not so good, I think that we can ignore it.